common/utils/src/main/scala/org/apache/spark/util/JsonUtils.scala

```diff
@@ -39,3 +39,5 @@ private[spark] object JsonUtils {
     new String(baos.toByteArray, StandardCharsets.UTF_8)
   }
 }
+
```

Comment:
hmm... should we always close `generator` and `baos` in `finally` block(line 37&38)? If so, we can refactor it in this pr

Reply:
And I want to know within what scope this `mapper` will be used in design?

Reply:
`ByteArrayOutputStream` does not really need to be closed.

Reply:
Yeah, it's not important, but it's a habit ... I may be a bit OCD :)

Reply:
Some OCD is fine :). In this particular case you have the concrete implementation of the InputStream, it is a stable implementation, and its semantics are very well understood. I am fine with omitting the close because of this.

Reply:
```
    /**
     * Closing a {@code ByteArrayOutputStream} has no effect. The methods in
     * this class can be called after the stream has been closed without
     * generating an {@code IOException}.
     */
    public void close() throws IOException {
    }
```

I guess probably we do not need to close (and this is existing code but just refactor anyway)